### PR TITLE
The M59A2 DMR for the TGMC

### PR DESCRIPTION
**Pull request by Hughgent**
## About The Pull Request

This is a teaching PR to show how to make guns

https://youtu.be/o-OvS4tC0N4

## Why It's Good For The Game

This will show people how to make new guns for the TGMC. thus allowing people to add stuff.

## Changelog
:cl: Hughgent
add: The M59A2 is now available for the marines to blast xenos with.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/updated_projectiles/magazines/rifles.dm [hunk 1]
```diff
@@ -33,6 +33,15 @@
 	icon_state = "m41a1_ap"
 	default_ammo = /datum/ammo/bullet/rifle/ap
 
+/obj/item/ammo_magazine/riflemarksman
+	name = "\improper M59A2 magazine (308)"
+	desc = "A 308 DMR magazine."
+	caliber = "minigun bullet"
+	icon_state = "m59a2"
+	w_class = WEIGHT_CLASS_NORMAL
+	default_ammo = /datum/ammo/bullet/minigun
```
Comment (Rohesie) [inline code/modules/projectiles/updated_projectiles/magazines/rifles.dm]: Perhaps it could use a bullet kind of its own?

**Comment by Rohesie** [comment]
It's lacking magazine sprites as well, right?

**Comment by Hughgent** [comment]
I made this purely for the video. I should put it on the wiki at some point.

I'm never going to finish this. I have left that as an exercise for the watcher.

**Comment by Rohesie** [comment]
A shame, as a DMR wouldn't be a terrible idea.

**Pull request closed without merging**